WordPress/Classes/Models/Revisions/DiffAbstractValue.swift

```diff
@@ -0,0 +1,42 @@
+import Foundation
+import CoreData
+
+
+class DiffAbstractValue: NSManagedObject {
+    enum Operation: String {
+        case add
+        case copy
+        case del
+        case unknown
+    }
+
+    enum DiffType: String {
+        case title
+        case content
+        case unknown
+    }
+
+    @NSManaged private var diffOperation: String
+    @NSManaged private var diffType: String
+
+    @NSManaged var index: Int
```

Comment:
Ah, yeah it looks like the new `index` property didn't make it into the model editor. Or maybe the intent wasn't for it to be managed?

Reply:
🤔It should be in the model editor. I can see it.
<img width="1269" alt="screenshot 2018-11-01 at 10 48 06" src="https://user-images.githubusercontent.com/912252/47847688-f004ae80-ddc3-11e8-8519-960419ef8ded.png">

Reply:
Also check this [commit](https://github.com/wordpress-mobile/WordPress-iOS/pull/10368/commits/d07e8f3704a5993ad8c5ed2c109c14830da34ccb). You can see it as attribute of that model.

Reply:
That's so weird. I refreshed the branch and no I see it too.
<img width="613" alt="screen shot 2018-11-01 at 9 32 34 am" src="https://user-images.githubusercontent.com/1435271/47858055-2852bf80-ddb9-11e8-8c5c-518a1fc9a735.png">